File: Iniciante/2721/2721.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.trim();

var nomes = [
  "Dasher",
  "Dancer",
  "Prancer",
  "Vixen",
  "Comet",
  "Cupid",
  "Donner",
  "Blitzen",
  "Rudolph",
];

var total = 0;

// lines.split(" ").forEach((curr) => {
//   total += parseInt(curr);
// });
//
for (let i = 0; i <= lines.length; i++) {
  let number = parseInt(lines[i]);
  if (!isNaN(number)) total += number;
}

var result = total % 9;

if (result === 0) result = 9;

console.log(nomes[result - 1]);

File: Iniciante/2721/2721.java
import java.util.Scanner;

public class Main {
  final static String[] NAMES = {
      "Dasher",
      "Dancer",
      "Prancer",
      "Vixen",
      "Comet",
      "Cupid",
      "Donner",
      "Blitzen",
      "Rudolph"
  };

  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    String input = scanner.nextLine();

    int total = 0;

    for (int i = 0; i < input.length(); i++) {
      char character = input.charAt(i);
      if (character == ' ')
        continue;
      total += character - '0';
    }

    int index = total % 9;
    if (index == 0)
      index = 9;

    System.out.println(NAMES[index - 1]);
  }
}
